larq_zoo/dorefanet.py

```diff
@@ -0,0 +1,183 @@
+import tensorflow as tf
+import larq as lq
+from larq_zoo import utils
+from zookeeper import registry, HParams
+
+
+@registry.register_model
+def dorefa_net(hparams, input_shape, num_classes, input_tensor=None, include_top=True):
+    def conv_block(x, features, kernel_size, strides=1, pool=False, padding="same"):
+        x = lq.layers.QuantConv2D(
+            features,
+            kernel_size=kernel_size,
+            strides=strides,
+            padding="same",
```

Comment:
Does this intentionally not use the padding argument of the function [here](https://github.com/larq/zoo/pull/75/files#diff-5bee5daf933f45f505789d9c9a479a59R9)?

Reply:
Oh no... This is not intentionally and means that the valid padding in the one layer is not used. I would suggest to change it to same padding in the model and put make it use the argument. Otherwise I will have to rerun the entire experiment I fear.

Reply:
The argument is used in the [MaxPool2D](https://github.com/larq/zoo/pull/75/files/35e449002442b8dbab650eadf7e1dd82c6c84d28#diff-5bee5daf933f45f505789d9c9a479a59R24) though.

>  I would suggest to change it to same padding in the model and put make it use the argument

Could you clarify?

Reply:
Yeah, the argument should indeed be called and I am using 'valid' padding later in the model (which is ignored), so if I just make it call the argument the model and the pretrained model do not fit anymore. So I would make it call the padding argument and change the 'valid' padding in the one layer to 'same'. Is this more clear?